src/test/kotlin/lotto/domain/NumbersTest.kt

```diff
@@ -0,0 +1,89 @@
+package lotto.domain
+
+import lotto.domain.number.BonusNumber
+import lotto.domain.number.Numbers
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class NumbersTest {
+
+    @Test
+    fun `make sure the numbers are not duplicated`() {
+        Assertions.assertThatCode { Numbers.fromInts(1, 2, 3, 4, 5, 6) }
+            .doesNotThrowAnyException()
+    }
+
+    @Test
+    fun `throw exception if numbers are duplicate`() {
+        Assertions.assertThatThrownBy { Numbers.fromInts(1, 2, 3, 4, 5, 5) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
+
+    @Test
+    fun `the selection must include 6 winning numbers`() {
+        Assertions.assertThatCode { Numbers.fromInts(1, 2, 3, 4, 5, 6) }
+            .doesNotThrowAnyException()
+    }
+
+    @Test
+    fun `throw exception if selection numbers are not included 6`() {
+        Assertions.assertThatThrownBy { Numbers.fromInts(1, 2, 3, 4, 5, 6, 7) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+    }
```

Comment:
```suggestion
    @Test
    fun `throw exception if selection numbers are not included 6`() {
        assertThrows<IllegalArgumentException>("Winning numbers must be $WINNING_NUMBER_COUNT unique values") {
            Numbers.fromInts(1, 2, 3, 4, 5, 6, 7)
        }
    }
```

This one might be easier + you can also validate exception message